### DISCUSSION TITLE
Where Can I Find the RefindPlus.efi File?

### DISCUSSION THREAD

**Discussion by btoland** [dakanji/RefindPlus#84] [General]
Hi,

I have an old mac (2008 iMac 24) on which I've installed linux mint and refind.  I'm looking for the refindplus.efi file so I can drop it in as a replacement for the refind.efi file as mentioned in your installation instructions.  I'm having video card issues (nvidia) and your project was mentioned on the refind site as having options that might help.

Any chance you could provide the refindplus.efi file or is it a specific build for different hardware??

Cheers,
-Ben

**Comment by dakanji** [comment]
You get such assets on GitHub via the link to the releases on the project page: https://github.com/dakanji/RefindPlus/releases

**Comment by btoland** [comment]
My bad!  Found it!
Many thanks for the quick reply.
